finalPrep/README.md

```diff
@@ -18,37 +22,45 @@ Some things to consider as you refactor include...
 
 Take your time as you go back and refactor your exercises from each section. We've included a handy checklist for you to go through below.
 
-- [ ] I have refactored my `section1` exercises to the best of my ability
-- [ ] I have refactored my `section2` exercises to the best of my ability
-- [ ] I have refactored my `section3` exercises to the best of my ability
-- [ ] I have refactored my `section4` exercises to the best of my ability
+- [x] I have refactored my `section1` exercises to the best of my ability
+- [x] I have refactored my `section2` exercises to the best of my ability
+- [x] I have refactored my `section3` exercises to the best of my ability
+- [x] I have refactored my `section4` exercises to the best of my ability
 
 ### Time Management Prep
 In Mod 0 you've learned about different techniques for managing your time at Turing. Please create a calendar for your **first 3 weeks of Mod 1**. Feel free to make your calendar fit your style, but we suggest that your calendar should include the following:
-- [ ] Standard M1 class schedule (see M1 calendar [here](https://frontend.turing.io/today/))
-- [ ] Study/Project work time
-- [ ] Health + Wellness
+<<<<<<< HEAD
+- [x] Standard M1 class schedule (see M1 calendar [here](https://frontend.turing.io/today/))
+- [x] Study/Project work time
+- [x] Health + Wellness
 
 When you are finished, add screenshots of your calendar so we can provide feedback if needed!
 
 - `Add Week 1 Screenshot Here`
 - `Add Week 2 Screenshot Here`
 - `Add Week 3 Screenshot Here`
+=======
+
+- <img width="1029" alt="Screen Shot 2021-04-24 at 10 54 52 AM" src="https://user-images.githubusercontent.com/79541611/115970452-72433900-a510-11eb-862e-02acd1255cfc.png">
+
+- <img width="1049" alt="Screen Shot 2021-04-24 at 10 55 36 AM" src="https://user-images.githubusercontent.com/79541611/115970464-7d966480-a510-11eb-8230-a1cd106077a8.png">
+
+- <img width="1035" alt="Screen Shot 2021-04-24 at 10 55 50 AM" src="https://user-images.githubusercontent.com/79541611/115970470-838c4580-a510-11eb-8c3f-bfde8b683d7c.png">
```

Comment:
I like that you worked through and ensured you had time for school work and self care!